samples/README.md

```diff
@@ -76,4 +88,12 @@ A remark of pass(multiple) indicates the test was successful, but that in conver
 
 A remark of retired indicates the test resulted in finding that the specified 2525C SIDC has been formally retired by the SSMC, and therefore has no recommended 2525D equivalent.
 
-	jmsml.exe /xl="LegacyMappingTableCtoD"
\ No newline at end of file
+	jmsml.exe /xl="LegacyMappingTableCtoD"
+
+### Schemas ###
+
+Military feature attribute schemas, which consist of Fields and Subtypes, can be exported from JMMSL data.  These export files are then used with tools in the military features data repository to automatically build a military feature template geodatabase.
+
+    jmsml.exe /xschemas="./military_feature_schemas/"
+
+The test folder [here](../source/JointMilitarySymbologyLibraryCS/jmsml/test) contains the current baseline for these tables.  The export function performs a line by line comparison test between each line of each newly created csv file and each line of the corresponding current baseline file, and creates an annotated version of each file [here](./military_feature_schemas/comparison_test_results/), highlighting any differences.  A product engineer can use these test results to identify deltas in the output and ascertain if those deltas are legitimate and intended differences in versions of that output.
```

Comment:
The intent for this test was to highlight differences that, once we examine those differences and accept or reject them, would change the baseline and the files in the jmsml/test folder would then be replaced with new baseline files.  Thus, over time, and a couple of iterations, we should see fewer and fewer, if any, future differences.